src/factors/PartialPose3.jl

```diff
@@ -19,11 +19,21 @@ end
 
 PriorPose3ZRP(z::T1,rp::T2) where {T1 <: SamplableBelief, T2 <: SamplableBelief} = PriorPose3ZRP{T1,T2}(z, rp)
 
-getManifold(::PriorPose3ZRP) = ProductGroup(ProductManifold(TranslationGroup(1), SpecialOrthogonal(3)))
+# getManifold(::PriorPose3ZRP) = ProductGroup(ProductManifold(TranslationGroup(1), SpecialOrthogonal(3)))
+getManifold(::PriorPose3ZRP) = SpecialEuclidean(3)
```

Comment:
yes, this should work when we include information `PriorPose3ZPR.partial`.  I suspect we will need to add more info defining which coordinates to use, in this case Euler angles rather than Lie Exponentials (so(3)).